### PR TITLE
Final Project Part One

### PR DESCRIPTION
**Pull request by AlizaBuchen**
API implementation

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -12,6 +12,7 @@ repositories {
 dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
+    //implementation 'com.AlizaBuchen:apikeys:0.2'
```
Comment (andrewoid) [inline build.gradle]: Why is your name here?

Comment (AlizaBuchen) [reply build.gradle]: I was following what was on your website. by dependencies you wrote 
implementation 'com.andrewoid:apikeys:0.2'

I just realized that it was your package name, I thought it had to be your gitHub username... but now my build is failing

Comment (andrewoid) [reply build.gradle]: You are still not following what I wrote. 

I never asked you to replace it with your name. 

It's a library that I wrote that you are using.

File: src/main/java/buchen/rijks/RijksService.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package buchen.rijks;
+
+import io.reactivex.rxjava3.core.Observable;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface RijksService {
+    @GET("/api/en/collection")
+    Observable<ArtObjects> page(
```
Comment (andrewoid) [inline src/main/java/buchen/rijks/RijksService.java]: `Single` not `Observable`

File: src/test/java/buchen/rijks/RijksServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        // when
+        ArtObjects collection = service.query(
+                apiKey.get(), 1, "e"
+        ).blockingGet();
+
+        // then
+        ArtObject artObject = collection.artObjects[0];
+        assertNotNull(artObject.title);
+        assertNotNull(artObject.longTitle);
+        assertNotNull(artObject.webImage);
```
Comment (andrewoid) [inline src/test/java/buchen/rijks/RijksServiceTest.java:47]: Assert that web image has a url

**Review by andrewoid — APPROVED** [review]

**Pull request merged**